### PR TITLE
Adding threshold for different parameters within gptp

### PR DESCRIPTION
**Pull request by davidcemin**
This patch fixes both Issues #136 and #137.

In both cases, a new configuration item was added to the .ini file.

### CODE REVIEW

File: daemons/gptp/common/avbts_port.hpp [hunk 1]
```diff
@@ -234,6 +234,9 @@ class IEEE1588Port {
 	static const int64_t INVALID_LINKDELAY = 3600000000000;
 	static const int64_t NEIGHBOR_PROP_DELAY_THRESH = 800;
 	static const unsigned int SYNC_RECEIPT_THRESH = 5;
+	static const unsigned int SEQID_ASCAPABLE_THRESHOLD = 5;
+	static const uint16_t LOSTPDELAY_RESP_THRESH = 3;
```
Comment (andrew-elder) [inline daemons/gptp/common/avbts_port.hpp]: Why aren't these called DEFAULT_SEQID_ASCAPABLE_THRESHOLD etc.?
They are really just initial values, correct?
They are overridden by the .ini file - at least that is what I assume.

File: daemons/gptp/common/avbts_port.hpp [hunk 2]
```diff
@@ -243,6 +246,14 @@ class IEEE1588Port {
 	unsigned int sync_receipt_thresh;
 	unsigned int wrongSeqIDCounter;
 
+    /*SeqID threshold*/
```
Comment (andrew-elder) [inline daemons/gptp/common/avbts_port.hpp]: indent

File: daemons/gptp/common/ptp_message.cpp [hunk 3]
```diff
@@ -1359,32 +1360,50 @@ void PTPMessagePathDelayRespFollowUp::processMessage(IEEE1588Port * port)
 	req->getPortIdentity(&req_id);
 	resp->getRequestingPortIdentity(&resp_id);
 
-	if (req->getSequenceId() != sequenceId) {
+    /*We need to keep track of lost respFUPs. If we have more
```
Comment (andrew-elder) [inline daemons/gptp/common/ptp_message.cpp]: check indent

**Comment by andrew-elder** [comment]
Overall looks good to me.

File: daemons/gptp/common/avbts_port.hpp (hunk 1 above)
Comment (davidcemin) [inline daemons/gptp/common/avbts_port.hpp]: Yes, I can change the name and they are overridden by the .ini file if a .ini file is used.

File: daemons/gptp/common/ptp_message.cpp (hunk 3 above)
Comment (davidcemin) [inline daemons/gptp/common/ptp_message.cpp]: This file mostly uses tabs, so i will keep tabs instead of spaces.

**Pull request merged**